full-backend-tests/src/test/java/org/graylog/plugins/views/SearchMetadataIT.java

```diff
@@ -93,6 +105,7 @@ void testRetrievingMetadataForStoredSearchWithoutParameter() {
 
     @ContainerMatrixTest
     void testRetrievingMetadataForStoredSearchWithParameter() {
+        LOG.error("testRetrievingMetadataForStoredSearchWithParameter()");
```

Comment:
Should we remove this logging before merging?

Reply:
Thanks for the review - I already removed those logging messages.